pyproject.toml

```diff
@@ -0,0 +1,66 @@
+[tool.poetry]
+name = "supervision"
+version = "0.12.0"
+description = "A set of easy-to-use utils that will come in handy in any Computer Vision project"
+authors = ["Piotr Skalski <piotr.skalski92@gmail.com>"]
+maintainers = ["Piotr Skalski <piotr.skalski92@gmail.com>"]
+readme = "README.md"
+packages = [{include = "supervision"}]
+homepage = "https://github.com/roboflow/supervision"
+repository = "https://github.com/roboflow/supervision"
+documentation = "https://github.com/roboflow/supervision/blob/main/README.md"
+keywords = ["machine-learning", "deep-learning", "vision", "ML", "DL", "AI", "YOLOv5", "YOLOv8", "Roboflow"]
+
+classifiers=[
+    'Intended Audience :: Developers',
+    'Intended Audience :: Science/Research',
+    'License :: OSI Approved :: BSD License',
+    'Programming Language :: Python :: 3',
+    'Programming Language :: Python :: 3.8',
+    'Programming Language :: Python :: 3.9',
+    'Programming Language :: Python :: 3.10',
+    'Programming Language :: Python :: 3 :: Only',
+    'Topic :: Software Development',
+    'Topic :: Scientific/Engineering',
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "Topic :: Scientific/Engineering :: Image Recognition",
+    'Typing :: Typed',
+    'Operating System :: Microsoft :: Windows',
+    'Operating System :: POSIX',
+    'Operating System :: Unix',
+    'Operating System :: MacOS'
+]
+
+[tool.poetry.dependencies]
+python = ">=3.8,<3.12.0"
+numpy = "^1.20.0"
+matplotlib = "^3.7.1"
+pyyaml = "^6.0"
+pillow = "^10.0.0"
```

Comment:
If I want to solve this issue: https://github.com/roboflow/supervision/pull/180#issuecomment-1638625859

Do I just need to change this line into `pillow = "^8.4.0"`

Reply:
Partially yes but let me do this. I forgot sorry about that. I am sending fix quickly.

Reply:
Awesome @onuralpszr 🔥! Thanks a lot 🙏🏻